README.md

```diff
@@ -38,6 +38,12 @@ PADL currently supports python **3.7**, **3.8** and **3.9**.
 Python version >= **3.8** is preferred because creating and loading transforms (**not** execution) 
 can be slower in **3.7**.
 
+WARNING: PADL transforms do not work in the base Python Interpreter environment because we rely on
+the inspect module to find source code (used when saving PADL transforms). Unfortunately, the 
+source code typed at this interactive prompt is discarded as soon as it is parsed. 
+Therefore, we recommend using the iPython interpreter or Jupyter Notebooks for interactive sessions 
+because they do not have this problem. 
```

Comment:
iIthink we can remove 'because they do not have this problem. ' from 
`Therefore, we recommend using the IPython interpreter or Jupyter Notebooks for interactive sessions because they do not have this problem. ` 
and make it 
`Therefore, we recommend using the IPython interpreter or Jupyter Notebooks for interactive sessions.`
Also, 
`iPython` to `IPython`